service/plugin/src/main/java/com/solace/maas/ep/event/management/agent/plugin/command/model/CommandResult.java

```diff
@@ -16,5 +16,4 @@ public class CommandResult {
     private JobStatus status;
     private Map<String, Object> result;
     private List<Map<String, Object>> logs;
-    private List<Map<String, Object>> errors;
```

Comment:
We decided to remove errors from the response since we could use the log level in the logs to determine which logs are errors.